README.md

```diff
@@ -1,31 +1,159 @@
 <p align="center">
-  <img src="https://raw.githubusercontent.com/amazeeio/lagoon/master/docs/images/lagoon-logo.png" alt="The Lagoon logo is a blue hexagon split in two pieces with an L-shaped cut" width="40%">
+ <img src="./Lagoon_OG.png" alt="The Lagoon logo is a blue hexagon split in two pieces with an L-shaped cut" width="40%">
 </p>
 
-# amazee.io Lagoon - Docker Build and Deploy System for OpenShift & Kubernetes
+# Lagoon - the developer-focused application delivery platform for Kubernetes
+
+Lagoon solves what developers are dreaming about: A system that allows developers to locally develop their code and their services with Docker and run the exact same system in production. The same container images, the same service configurations and the same code.
+
+> As an application delivery **platform**, the primary focus of Lagoon is as a tool for the deployment, management, security and operation of multiple applications, utilising cloud-native technologies, but with minimal imposition or knowledge expectation on the developers of those applications.
```

Comment:
This paragraph is a little complex, I ran it through Hemingway and tried to simplify it - thoughts?
> Lagoon is an application delivery **platform**. Its primary focus   is as a tool for the deployment, management, security and operation of many applications. Lagoon uses cloud-native technologies, but with little knowledge needed by the application developers.